### DISCUSSION TITLE
Bug: CuatomDialog

### DISCUSSION THREAD

**Discussion by RoyTheReal** [Sketchware-Pro/Sketchware-Pro#975] [General]
In the newest UI redesign Sketchware debug snapshot as of writing this, I get an error with any project that has a CustomDialog, it doesn't happen in Sketchware Pro beta 5:
https://drive.google.com/file/d/1hgYcQlWxiNjBSc-bBzgw_yJQlwww_r-U/view?usp=drivesdk

**Comment by elfilibusterismo** [comment]
Try to check your custom component for CustomDialog

**Reply by RoyTheReal** [reply]
I did looks fine

**Comment by rukamori** [comment]
btw duplicate #973

**Comment by j-px** [comment]
Why did you create an Issue and a Discussion?

**Reply by RoyTheReal** [reply]
Sorry it just seems that people respond to discussions more than issues

**Reply by RoyTheReal** [reply]
Also I had the informational video to add but yeah sorry
